src/main/java/uk/gov/register/service/ItemValidator.java

```diff
@@ -43,25 +44,36 @@ private void validateFields(JsonNode inputEntry, RegisterMetadata registerMetada
         throwEntryValidationExceptionIfConditionIsFalse(!unknownFields.isEmpty(), inputEntry, "Entry contains invalid fields: " + unknownFields.toString());
     }
 
-    private void validateFieldsValue(JsonNode inputEntry, Map<String, Field> fields) throws ItemValidationException {
+    private void validateFields(final JsonNode inputEntry, final Map<String, Field> fields) throws ItemValidationException {
         inputEntry.fieldNames().forEachRemaining(fieldName -> {
-            Field field = fields.get(fieldName);
+            final Field field = fields.get(fieldName);
+            final Datatype datatype = field.getDatatype();
+            final JsonNode fieldValue = inputEntry.get(fieldName);
 
-            Datatype datatype = field.getDatatype();
-
-            JsonNode fieldValue = inputEntry.get(fieldName);
+            this.validateFieldDataType(fieldName,datatype, inputEntry, fieldValue);
+            this.validateFieldValue(fieldName, field, datatype, inputEntry, fieldValue);
+        });
+    }
 
-            if (field.getCardinality().equals(Cardinality.MANY)) {
+    private void validateFieldDataType(final String fieldName, final Datatype datatype, final JsonNode inputEntry, final JsonNode fieldValue) {
```

Comment:
Now that you have implemented `DatetimeDatatype.isValid()` I don't think you need this new method called `validateFieldDataType` as `Datatype.isValid()` is an interface method that gets called for all Datatypes (including StringDatatype, IntegerDatatype etc.) in the original existing `validateField` method. I don't think this class requires any changes at all.